### PR TITLE
Updated Polish translation, icon and Privacy Policy.

### PR DESCRIPTION
**Pull request by pioskala**
Updated Polish translation, icon and Privacy Policy.

### CODE REVIEW

**Comment by misi** [comment]
Let's do one thing in one PR. If you split it by function well be the better..

**Comment by pioskala** [comment]
Hmm.. not sure how to split it. Most of it are translations, then index.html, icons and privacy policy. Maybe this time you will be able to merge it and next time I'll do it in smaller parts?

**Comment by misi** [comment]
I will leave it to @havfo he is the captain..

File: app/public/index.html [hunk 1]
```diff
@@ -10,10 +10,28 @@
...
+<link rel="apple-touch-icon" sizes="152x152" href="/apple-icon-152x152.png">
+<link rel="apple-touch-icon" sizes="180x180" href="/apple-icon-180x180.png">
+<link rel="icon" type="image/png" sizes="192x192"  href="/android-icon-192x192.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="96x96" href="/favicon-96x96.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+<link rel="manifest" href="/manifest.json">
+<meta name="msapplication-TileColor" content="#ffffff">
+<meta name="msapplication-TileImage" content="/ms-icon-144x144.png">
+<meta name="theme-color" content="#ffffff">
```
Comment (havfo) [inline app/public/index.html:31]: Duplicate entry, see line 10

File: app/public/index.html [hunk 2]
```diff
@@ -10,10 +10,28 @@
 		<meta name='theme-color' content='#000000' />
 
 		<link rel='preconnect' href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
-		<link rel='shortcut icon' href='%PUBLIC_URL%/favicon.ico' />
-		<link rel='manifest' href='%PUBLIC_URL%/manifest.json' />
+<!--		<link rel='shortcut icon' href='%PUBLIC_URL%/favicon.ico' />
+		<link rel='manifest' href='%PUBLIC_URL%/manifest.json' />-->
+<link rel="apple-touch-icon" sizes="57x57" href="/apple-icon-57x57.png">
```
Comment (havfo) [inline app/public/index.html:15]: Please fix double quotes to single quotes on all lines.

File: app/public/index.html [hunk 3]
```diff
@@ -10,10 +10,28 @@
...
+<link rel="icon" type="image/png" sizes="96x96" href="/favicon-96x96.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+<link rel="manifest" href="/manifest.json">
+<meta name="msapplication-TileColor" content="#ffffff">
+<meta name="msapplication-TileImage" content="/ms-icon-144x144.png">
+<meta name="theme-color" content="#ffffff">
 
-		<title>Multiparty Meeting</title>
+
+		<title>Web conferencing service</title>
```
Comment (havfo) [inline app/public/index.html:34]: This is wrong, please fix.

File: app/public/browserconfig.xml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig><msapplication><tile><square70x70logo src="/ms-icon-70x70.png"/><square150x150logo src="/ms-icon-150x150.png"/><square310x310logo src="/ms-icon-310x310.png"/><TileColor>#ffffff</TileColor></tile></msapplication></browserconfig>
```
Comment (havfo) [inline app/public/browserconfig.xml:2]: Please format this file properly

File: app/public/manifest.json [hunk 5]
```diff
@@ -1,15 +1,41 @@
...
-		"sizes": "64x64 32x32 24x24 16x16",
-		"type": "image/x-icon"
-		}
-	],
-	"start_url": ".",
-	"display": "standalone",
-	"theme_color": "#000000",
-	"background_color": "#ffffff"
-}
+ "name": "App",
```
Comment (havfo) [inline app/public/manifest.json:2]: This manifest file is missing short_name and has a wrong name.

File: app/public/privacy/privacy.html [hunk 6]
```diff
@@ -1,18 +1,78 @@
...
 </head>
 <body>
-	<header><h1>Privacy Statement</h1></header>
-	<h2>Privacy Policy</h2>
-	<ul>
-		<li>User consent</li>
-		<li>Data deletion</li>
-	</ul>
-	<h2>Acceptable use policy (AUP)</h2>
+<h1>eduMeet &ndash; Privacy Notice</h1>
```
Comment (havfo) [inline app/public/privacy/privacy.html:6]: This is not the proper place for edumeet specific instance privacy notice. Geant is not responsible for instances forked from this repository, nor do we want to be contacted by the users of various instances. This needs to only be a skeleton, please revert.

File: package-lock.json [hunk 7]
```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```
Comment (havfo) [inline package-lock.json:2]: Please remove this file.

File: app/public/index.html (hunk 2 above)
Comment (havfo) [inline app/public/index.html:15]: Please fix indentation on all lines

File: app/public/index.html [hunk 8]
```diff
@@ -10,10 +10,28 @@
...
+<link rel="apple-touch-icon" sizes="114x114" href="/apple-icon-114x114.png">
+<link rel="apple-touch-icon" sizes="120x120" href="/apple-icon-120x120.png">
+<link rel="apple-touch-icon" sizes="144x144" href="/apple-icon-144x144.png">
+<link rel="apple-touch-icon" sizes="152x152" href="/apple-icon-152x152.png">
+<link rel="apple-touch-icon" sizes="180x180" href="/apple-icon-180x180.png">
+<link rel="icon" type="image/png" sizes="192x192"  href="/android-icon-192x192.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="96x96" href="/favicon-96x96.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+<link rel="manifest" href="/manifest.json">
```
Comment (havfo) [inline app/public/index.html:28]: Duplicate entry, see line 14

File: app/public/index.html (hunk 2 above)
Comment (havfo) [inline app/public/index.html:15]: Please use the proper %PUBLIC_URL% syntaxt for importing assets.

**Comment by pioskala** [comment]
> I have gone over most of it now. Next time, please split this into smaller PR.

Thanks Havar, next time will make pull request for smaller changes.

**Comment by pioskala** [comment]
Made all requested changes. Not sure now if I should close this and reopen or add here somehow.

**Pull request closed without merging**